modules/table/src/test/java/org/apache/ignite/internal/table/distributed/raft/snapshot/PartitionSnapshotStorageFactoryTest.java

```diff
@@ -44,24 +39,18 @@
  */
 @ExtendWith(MockitoExtension.class)
 public class PartitionSnapshotStorageFactoryTest {
-    private final MvPartitionStorage mvPartitionStorage = new TestMvPartitionStorage(0);
-    private final TxStateStorage txStateStorage = new TestTxStateStorage();
-
     @Mock
     private PartitionAccess partitionAccess;
 
-    @BeforeEach
-    void configureMocks() {
-        when(partitionAccess.mvPartitionStorage()).thenReturn(mvPartitionStorage);
-        when(partitionAccess.txStatePartitionStorage()).thenReturn(txStateStorage);
-    }
-
     @Test
     void testForChoosingMinimumAppliedIndexForMeta() {
-        mvPartitionStorage.lastApplied(10L, 2L);
-        txStateStorage.lastApplied(5L, 1L);
+        lenient().when(partitionAccess.minLastAppliedIndex()).thenReturn(5L);
```

Comment:
These `lenient()`s seem redundant (as the corresponding methods are always called under this test), so they should be dropped.

Reply:
I tried to delete it, but got the following error: `Please remove unnecessary stubbings or use 'lenient' strictness. More info: javadoc for UnnecessaryStubbingException class.`

Reply:
Fix it